FirebaseInAppMessaging/Sources/FIRInAppMessagingPrivate.h

```diff
@@ -18,9 +18,13 @@
 #import "FIRCore+InAppMessaging.h"
 #import "FIRInAppMessaging.h"
 
+@class FIRInstallations;
+
 @protocol FIRInAppMessagingInstanceProvider;
 @protocol FIRLibrary;
+@protocol FIRAnalyticsInterop;
 
 @interface FIRInAppMessaging () <FIRInAppMessagingInstanceProvider, FIRLibrary>
 @property(nonatomic, readwrite, strong) id<FIRAnalyticsInterop> _Nullable analytics;
+@property(nonatomic, readwrite, strong) FIRInstallations* _Nonnull installations;
```

Comment:
Technically it is `nullable`. 

Another question - what's the difference between specifying nullability inside property (like `@property(nullable, strong) ...` and  current `_Nullable` annotation?

Reply:
There's no difference (see 2nd section in this [doc](https://developer.apple.com/swift/blog/?id=25))-- but I changed this to nullable and handled the nil case in 72f47b6.